international-journal-of-geriatric-psychiatry.csl

```diff
@@ -20,7 +19,7 @@
     <issn>0885-6230</issn>
     <eissn>1099-1166</eissn>
     <summary>Harvard (author-date style) for International Journal of Geriatric Psychiatry</summary>
```

Comment:
we should add a note that this follows published articles and not the (useless) author guidelines